strawberryfields/ops.py

```diff
@@ -859,23 +874,12 @@ def _apply(self, reg, backend, **kwargs):
         backend.prepare_ket_state(ket, *reg)
 
 
-class GKPstate(Preparation):
-    r"""Prepare a mode in a GKP state.
-
-    .. warning::
```

Comment:
why is this docstring being shortened?

Reply:
In anticipation of replacement prep classes from a different branch